packages/amp/README.md

```diff
@@ -0,0 +1,71 @@
+# `@frontity/amp`
+
+## Table of contents
+
+<!-- toc -->
+
+- [Install](#install)
+- [Usage](#usage)
+- [Feature Discussions](#feature-discussions)
+- [Changelog](#changelog)
+- [Open Source Community](#open-source-community)
+  - [Channels](#channels)
+  - [Get involved](#get-involved)
+
+<!-- tocstop -->
+
+## Install
+
+```sh
+npm i @frontity/amp
+```
+
+## Usage
+
+You can should install the package and add it to your `frontity.settings.js` file:
+
+```js
+export default {
+  packages: [
+    "@frontity/amp",
+    // ...your other packages
+  ],
+};
```

Comment:
Maybe we can say here a bit more about how to normally set up two sites, one for React and another for AMP.

```suggestion
You should add the package to your `frontity.settings.js` file.

Usually, you will want to have a "main site" which doesn't use this package and renders the normal React application and an "amp site" that returns the AMP version of the pages.

The AMP site should have a `match` property which will indicate which are the AMP URLs. Some examples are:

- A subdomain: `"match": "amp\\.domain\\.com"`
- A query: `"match": "(\\?|&)amp=true"`
- An ending path: `"match": "\\/amp\\/?($|\\?|#)"`

You can read more about setting up multiple sites [in our docs](https://docs.frontity.org/learning-frontity/settings#multiple-sites).

```js
export default [
  {
    name: "main-site",
    packages: [
      // ...your normal packages
    ],
  },
  {
    name: "amp-site",
    match: "...",
    packages: [
      "@frontity/amp",
      // ...your other packages
    ],
  },
];
```